### DISCUSSION TITLE
Can we have datalist element feature?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [chakra-ui/chakra-ui#6112] [Ideas]
I was working on some web application and there I was requested the feature of getting predictions while taking input, this can be done normally but there seems no way to style the pop-over section(options) in normal datalist element. I thought of using Select element but it won't have typing facility. I thought of creating a new feature request but I didn't knew the proposed idea how to implement it.

**Comment by mariosantosdev** [comment]
Might you can use it [chakra-ui-autocomplete](https://github.com/koolamusic/chakra-ui-autocomplete)
